src/turnkeyml/run/onnxrt/execute.py

```diff
@@ -9,9 +9,11 @@
 import json
 from statistics import mean
 import platform
+import onnxruntime
 import turnkeyml.run.plugin_helpers as plugin_helpers
 
-ORT_VERSION = "1.15.1"
+# Use the same ORT version of the base environment in the new conda env
+ORT_VERSION = onnxruntime.__version__
```

Comment:
What's the point of having a within_conda.py for OnnxRT if it's always the same onnxruntime version as the base env? On second thought maybe dont answer that...